Components/Image/image.html

```diff
@@ -0,0 +1,131 @@
+				</p>
+
+				<br />
+
+				<div class="img-container">
+					<img
+						src="https://plchldr.co/i/300x300?&amp;bg=ccc&amp;fc=424242&amp;text=Responsive%20Image(Currently%20300*300px)"
+						class="img-responsive"
+						alt="Responsive Image"
+					/>
+				</div>
+				<br />
+				<iframe
+					src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=seti&wt=sharp&l=htmlmixed&width=522.25&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=15.5px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%2520src%253D%2522...%2522%2520class%253D%2522img-responsive%2522%2520alt%253D%2522...%2522%253E"
+					style="
+						width: 522px;
+						height: 95px;
+						border: 0;
+						transform: scale(1);
+						overflow: hidden;
+					"
+					sandbox="allow-scripts allow-same-origin"
+				>
+				</iframe>
+
+				<hr />
+			</div>
+
+			<!-- Rounded Corners Images -->
+
+			<div class="component-info-div">
+				<h3 class="text-title">Rounded Corners Images</h3>
+				<p class="text-description">
+					The <code>".img-rounded-corners"</code> class adds rounded corners to
+					an image.
+				</p>
+
+				<br />
+
+				<div class="img-container">
+					<img
+						src="https://plchldr.co/i/200x200?&amp;bg=ccc&amp;fc=424242&amp;text=Rounded%20Corners"
+						class="img-rounded-corners"
+						alt="Rounded Corners Image"
+					/>
+				</div>
+				<br />
+				<iframe
+					src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=seti&wt=sharp&l=htmlmixed&width=599.75&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=15.5px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%2520src%253D%2522...%2522%2520class%253D%2522img-rounded-corners%2522%2520alt%253D%2522...%2522%253E"
+					style="
+						width: 600px;
+						height: 95px;
+						border: 0;
+						transform: scale(1);
+						overflow: hidden;
+					"
+					sandbox="allow-scripts allow-same-origin"
+				>
+				</iframe>
+				<hr />
+			</div>
+
+			<!-- Circle Images -->
+
+			<div class="component-info-div">
+				<h3 class="text-title">Circle Images</h3>
+				<p class="text-description">
+					The <code>".img-circle"</code> class shapes the image to cicle.
+				</p>
+
+				<br />
+
+				<div class="img-container">
+					<img
+						src="https://plchldr.co/i/200x200?&amp;bg=ccc&amp;fc=424242&amp;text=Circle%20Image"
+						class="img-circle"
+						alt="Circle Image"
+					/>
+				</div>
+				<br />
+				<iframe
+					src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=seti&wt=sharp&l=htmlmixed&width=522.25&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=15.5px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%2520src%253D%2522...%2522%2520class%253D%2522img-circle%2522%2520alt%253D%2522...%2522%253E"
+					style="
+						width: 522px;
+						height: 95px;
+						border: 0;
+						transform: scale(1);
+						overflow: hidden;
+					"
```

Comment:
This is kind of repetitive, we use 2-3 classes for better readability

Reply:
This is the default iframe code that I copied from carbon.now.sh
Yes I can add styling classes, but then I would I have to manually add those classes every time. This is just for the documentation purposes.
Thank you for kind inputs.